x-pack/plugins/ingest_manager/server/services/output.ts

```diff
@@ -133,6 +141,12 @@ class OutputService {
       perPage: 1000,
     };
   }
+
+  // Warning! This method is not going to working in a scenario with multiple Kibana instances,
+  // in this case Kibana should be restarted if the Admin User change
+  public invalidateCache() {
+    cachedAdminUser = null;
+  }
 }
```

Comment:
What would be the behavior when this happens? I presume we would raise an error in Kibana?

Reply:
Yes we are going to have errors when enrolling new agents